rubygem-smart_proxy_dynflow_core/rubygem-smart_proxy_dynflow_core.spec

```diff
@@ -83,8 +84,9 @@ cp -pa .%{_bindir}/* %{buildroot}%{root_bindir}/
 mkdir -p %{buildroot}%{root_sysconfdir}/smart_proxy_dynflow_core
 cp -pa .%{gem_instdir}/config/settings.yml.example %{buildroot}%{root_sysconfdir}/smart_proxy_dynflow_core/settings.yml
 
-#copy init scripts and sysconfigs
+#copy init scripts, sysconfigs and logrotate config
 install -Dp -m0644 %{buildroot}%{gem_instdir}/deploy/%{service_name}.service %{buildroot}%{_unitdir}/%{service_name}.service
+install -Dp -m0644 %{SOURCE1} %{buildroot}%{_sysconfdir}/logrotate.d/%{name}
```

Comment:
Is `_sysconfdir` correct in an SCL environment? Wouldn't this be within the SCL root? Is that supported?